src/SFA.DAS.Charities.Database/Tables/Charity.sql

```diff
@@ -0,0 +1,24 @@
+﻿CREATE TABLE [dbo].[Charity]
+(
+    [Id] INT NOT NULL PRIMARY KEY, 
+    [Name] VARCHAR(MAX) NOT NULL, 
```

Comment:
Noticed that you have no indexing on this table. Unless you're searching by Primary Key then this could be a little slow

Reply:
It is going to use the primary key.